gen-resourcesdocs/pkg/outputs/interface.go

```diff
@@ -28,7 +28,7 @@ type Chapter interface {
 // Section is an interface to a section of an output
 type Section interface {
 	AddContent(s string) error
-	AddTypeDefinition(s string) error
+	AddTypeDefinition(typ string, description string) error
```

Comment:
nit: `typ` is the type string?

Reply:
Yes, as `type` is a reserved keyword in Go, we cannot use `type` as variable name

Reply:
Right, `typ` is odd. `t` or something else might make it more readable.

Reply:
I think this is quite common in Go, see for example https://golang.org/pkg/go/types/

Reply:
Ha, interesting.